src/pages/cli/start/install.mdx

```diff
@@ -19,6 +19,10 @@ import all0 from "/src/fragments/cli-install-block.mdx";
 - Verify that you are running at least Node.js version 12.x and npm version 6.x or greater by running `node -v` and `npm -v` in a terminal/console window
```

Comment:
Dont we need to update that?

Reply:
If you mean bumping the Node / npm versions, we do not, because not all users of `amplify-js` / the CLI use CRA.